dev/io.openliberty.cdi.4.0.interfaces/bnd.bnd

```diff
@@ -0,0 +1,30 @@
+#*******************************************************************************
+# Copyright (c) 2017, 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=1.0
+	com.ibm.ws.cdi.extension
```

Comment:
The bnd file is missing the bundle name and symbolic name and the project is missing the eclipse project files.

I'll make a commit to fix both of these.